src/main/groovy/com/rllc/spreadsheet/service/ArchivedMinistersServiceImpl.groovy

```diff
@@ -0,0 +1,41 @@
+package com.rllc.spreadsheet.service
+
+import com.rllc.spreadsheet.dao.MinisterDAO
+import com.rllc.spreadsheet.domain.Minister
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+
+/**
+ * Created by Robert on 5/10/2015.
+ */
+class ArchivedMinistersServiceImpl implements ArchivedMinistersService {
+    private static final Logger logger = LoggerFactory.getLogger(ArchivedMinistersServiceImpl.class);
+
+    @Autowired
+    MinisterDAO ministerDAO
+
+    @Override
+    void updateSpreadsheet(List<Minister> ministers) {
```

Comment:
when will `updateSpreadsheet`  be used?
I was expecting a read-only connection to a data source to extract a list of ministers.